File: typescript/src/shapes/rectangle.ts
interface Rectangle {
    // width: number,
    // height: number,
    computeArea: () => number
}

function newRectangle(width: number, height: number): Rectangle {
        // int width,
        // int height,
    return {
        computeArea: function (): number {
            return width * height
        }
    }
}

export { Rectangle, newRectangle }

File: java/src/main/java/edu/cmu/cs/cs214/lab02/shapes/Rectangle.java
package edu.cmu.cs.cs214.lab02.shapes;

public class Rectangle implements Shape {
    private double height; // change to private
    private double width; // change to private
    
    public Rectangle(double height, double width){
        this.height = height;
        this.width = width;
    }

    public double getArea() {
        return height * width;
    }
}
